deploy/sample-backup-configs/sample_config_build.ini

```diff
@@ -12,56 +12,37 @@ postbuildhook=
 [builds]
 # this section references builds defined in config_build_recipes.ini
 # if you add a build here, it will be built when you run buildafi
-# Legacy recipe without an L2
 firesim-quadcore-nic-llc4mb-ddr3
```

Comment:
While we're renaming these, can we switch to the format
firesim-rocket-quadcore-nic... etc
and
firesim-boom-quadcore-nic... etc

i.e. firesim -> firesim-rocket and fireboom -> firesim-boom
?

Or even just get rid of firesim from these names (although one could imagine a different DESIGN).

Reply:
CC @davidbiancolin

Reply:
How about `chipyard-rocket-quadcore..etc`, `chipyard-boom-singlecore...etc`?

The naming for a non-default DESIGN would be `howietop-rocket-quadcore...etc`, for example.

Reply:
is DESIGN now chipyard?

Reply:
I believe DESIGN is actually still `firesim`, but that's just because we now do `class FireSim extends chipyard.Top`.

Reply:
What is the consensus here. `chipyard-rocket/boom` or `firesim-rocket/boom`?

Reply:
I think let's leave it correlated to DESIGN, so firesim-

We can revisit this after further refactoring/renaming, but it would be good for this to ultimately reflect the target design

Reply:
@davidbiancolin what do you think?

Reply:
Isn't the plan to move towards a universal Harness? In that case DESIGN is always FireSim. IMO that first word should be correlated with the type of the Top instantiated within the Harness, which is `chipyard.Top` in most cases.

Reply:
Let's leave it for now.

Reply:
Ok I'll change everything back to the original firesim/fireboom namings

Reply:
@jerryz123 i just spoke to @davidbiancolin. to clarify what we're saying is to go with:

firesim-boom-MORE-STUFF
firesim-rocket-MORE-STUFF

Reply:
Ok that's fine